Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -821,6 +823,54 @@ def create_stix_object(xsoar_indicator: dict, xsoar_type: str, extensions_dict:
     return stix_object, extension_definition, extensions_dict
 
 
+def build_sco_object(stix_object: Dict[str, Any], xsoar_indicator: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Can you add a unit test to this specific function?

Reply:
Yes, on it.